docs/audio/bluez-alsa.md

```diff
@@ -0,0 +1,111 @@
+# bluez-alsa
+
+## Introduction
+
+Bluetooth Audio ALSA Backend allow bluetooth ausio without PulseAudio.
```

Comment:
A quick general description of bluez-alsa could be useful (copied from github.com/Arkq/bluez-alsa). The fact that it allows to use BT audio without pulseaudio is specific to AGL and may not be the quickdesc to push first :)